<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>authentication</class>
 <widget class="QMainWindow" name="authentication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>authentication</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/img/logo.png</normaloff>:/img/img/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
 border-image:url(:/img/img/back.500d33d7f3b099cbb2d1.jpg);


  
    text-rendering: optimizeLegibility;
    overflow: hidden;
    touch-action: manipulation;
   
    font-family: -apple-system,BlinkMacSystemFont,&quot;Segoe UI&quot;,Roboto,&quot;Helvetica Neue&quot;,Arial,&quot;Noto Sans&quot;,&quot;Liberation Sans&quot;,sans-serif,&quot;Apple Color Emoji&quot;,&quot;Segoe UI Emoji&quot;,&quot;Segoe UI Symbol&quot;,&quot;Noto Color Emoji&quot;;
    font-size: 1rem;
    font-weight: 400;
    line-height: 1.5;
    color: #212529;
    text-align: left;
    background-color: #fff;

}

#frame{
 border-image:url(:/img/img/logolo.png);
position:fixed;
}



#label ,#label_2,#label_3,#Retourne_Accueil ,#commandLinkButton{
       
 color: #ffffff; 
 margin-top: 0;
 margin-bottom: 1rem

}


#pushButton
{
  color: #222428f;
  background-color: #f4f5f8;
  border-width: 0px;
  border-radius: 3px;
}



#topPanel { background-color: qlineargradient(spread:reflect, x1:0.5, y1:0, x2:0, y2:0, stop:0 rgba(46, 204, 233, 100), stop:1 rgba(32, 80, 96, 5)); 
}


#pushButton_Valider
{
  color: white;
  background-color: #e0b500;
  border-width: 0px;
  border-radius: 3px;
}

#pushButton_Inscrire:hover { background-color: #ffd31a; }

#topPanel { background-color: qlineargradient(spread:reflect, x1:0.5, y1:0, x2:0, y2:0, stop:0 rgba(46, 204, 233, 100), stop:1 rgba(32, 80, 96, 5)); 
}



</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="Nom_Complet">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>1321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string> Nom Complet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Valider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>1321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>VALIDER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Inscrivez-vous</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Tel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>1321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string> Téléphone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Email">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>1321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string> Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Code_Secret">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>1321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
    <property name="placeholderText">
     <string>Code Secret</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Confirmer_Code">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>1321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
    <property name="placeholderText">
     <string> Confirmer votre code Secret</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="Retourne_Accueil">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>590</y>
      <width>172</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>  Retourner à l'accueil ?</string>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
